client/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.controller.js

```diff
@@ -0,0 +1,54 @@
+angular.module('managerApp').controller('XdslAccessComfortExchangeCtrl', class XdslAccessComfortExchangeCtrl {
+  constructor($stateParams, $translate, OvhApiXdsl, TucToast, TucToastError,
+    COMFORT_EXCHANGE_TYPE_ERROR) {
+    this.$stateParams = $stateParams;
+    this.$translate = $translate;
+    this.OvhApiXdsl = OvhApiXdsl;
+    this.TucToast = TucToast;
+    this.TucToastError = TucToastError;
+    this.COMFORT_EXCHANGE_TYPE_ERROR = COMFORT_EXCHANGE_TYPE_ERROR;
```

Comment:
and then import them in this file and inject them here :)

Reply:
OK but it's not implemented like that in the telecom-manager.